porting/nimble/include/log/log.h

```diff
@@ -30,11 +30,13 @@ log_dummy(void *log, ...)
     (void)log;
 }
 
+#if 0
```

Comment:
why is this needed?

Reply:
Please see previous comment: https://github.com/apache/mynewt-nimble/pull/878#issuecomment-718964467

Reply:
Any suggestion on what to do about it? I think it is the only remaining issue in the PR

Reply:
Hmm I think for now we could just put all this under #ifndef LOG_DEBUG

in long term I think we should add own NPL_LOG_* and use those instead of relying on modlog module defining those

Reply:
I think that would require an include to be present before defining LOG_DEBUG which may not always be the case. What about `#if MYNEWT`?

Reply:
I think `#if MYNEWT` is ok for now, we'll need to resolve this anyway in some other way as @sjanc proposed